src/main/java/org/fcrepo/spec/testsuite/versioning/LdpcvHttpPost.java

```diff
@@ -54,14 +58,20 @@ public void ldpcvOfLdprsMustSupportPostWithoutMementoDatetimeHeader(final String
                                         "https://fcrepo.github.io/fcrepo-specification/#ldpcv-post",
                                         ps);
 
+        // create the versioned resource
+        final String newResource = createVersionedResource(uri, info).body().asString();
```

Comment:
I do not believe we can expect that other Fedora implementations will return the location of the created resource in the body of the response.
You should probably check the `Location` header.

Reply:
@bseeger : there's a getLocation() that should be available.